src/test/regress/sql/alter_database_propagation.sql

```diff
@@ -56,4 +48,64 @@ alter database regression set lock_timeout from current;
 alter database regression set lock_timeout to DEFAULT;
 alter database regression RESET lock_timeout;
 
+set citus.enable_create_database_propagation=on;
+create database regression2;
+alter database regression2 with CONNECTION LIMIT 100;
+alter database regression2 with IS_TEMPLATE true CONNECTION LIMIT 50;
+alter database regression2 with IS_TEMPLATE false;
+
+
+\set alter_db_tablespace :abs_srcdir '/tmp_check/ts3'
+CREATE TABLESPACE alter_db_tablespace LOCATION :'alter_db_tablespace';
+
+\c - - - :worker_1_port
+\set alter_db_tablespace :abs_srcdir '/tmp_check/ts4'
+CREATE TABLESPACE alter_db_tablespace LOCATION :'alter_db_tablespace';
+
+\c - - - :worker_2_port
+\set alter_db_tablespace :abs_srcdir '/tmp_check/ts5'
+CREATE TABLESPACE alter_db_tablespace LOCATION :'alter_db_tablespace';
+
+\c - - - :master_port
+
+set citus.log_remote_commands = true;
+set citus.grep_remote_commands = '%ALTER DATABASE%';
+
+alter database regression2 set TABLESPACE alter_db_tablespace;
```

Comment:
Can we add a test for SET TABLESPACE where dbname needs quoting, such as "regression!\'2"?